ideascube/models.py

```diff
@@ -25,8 +25,8 @@ class TimeStampedModel(models.Model):
     An abstract base class model that provides self-
     updating ``created_at`` and ``modified_at`` fields.
     """
-    created_at = models.DateTimeField(auto_now_add=True)
-    modified_at = models.DateTimeField(auto_now=True)
+    created_at = models.DateTimeField(_('creation date'), auto_now_add=True)
+    modified_at = models.DateTimeField(_('modification date'), auto_now=True)
```

Comment:
> I'd prefer you pass the verbose name using its keyword argument:

Agreed on the topic, but that's how is done for all the others fields, that would become inconsistent. I suggest to create a ticket to change this all in once in a separate move instead.